packages/engine-test-utils/src/__tests__/dendron-cli/commands/exportPodV2CLI.spec.ts

```diff
@@ -0,0 +1,156 @@
+          return;
+        },
+        { expect }
+      );
+    });
+  });
+
+
+
+  // dendron exportPodV2
+  describe("WHEN no config is given", () => {
+    test("THEN error must be thrown", async () => {
+      await runEngineTestV5(
+        async ({ wsRoot }) => {
+          const cmd = new ExportPodV2CLICommand();
+          const resp = await cmd.enrichArgs({
+            wsRoot
+          });
+          expect(resp.error?.severity).toEqual(ERROR_SEVERITY.FATAL);
+          return;
+        },
+        { expect }
+      );
+    });
+  });
+
+  // dendron exportPodV2 --inlineConfig Key=exportScope,Value=Vault --podId dendron.markdown
+  describe("WHEN both --podId and --inlineConfig is given", () => {
+    test("THEN inlineConfig values should be of higher precedence", async () => {
+      await runEngineTestV5(
+        async ({ wsRoot }) => {
+          const cmd = new ExportPodV2CLICommand();
+          const podId = "dendron.markdown";
+          const configPath = getCustomConfigPath(wsRoot, podId);
+          writeYAML(configPath, {
+            exportScope: "Workspace",
+            podType: PodV2Types.MarkdownExportV2,
+            destination: "workspace-exp"
+          });
+          const resp = await cmd.enrichArgs({
+            wsRoot,
+            inlineConfig: ["Key=exportScope,Value=Vault"],
+            podId,
+            vault: "vault1"
+          });
+          expect(resp.data?.config.exportScope).toEqual(PodExportScope.Vault);
+          expect(resp.data?.config.destination).toEqual("workspace-exp");
+          return;
+        },
+        { expect }
+      );
+    });
+  });
+
+    // dendron exportPodV2 --podId dendron.gdoc
+    describe("WHEN custom config has a connectionId is given", () => {
+      test("THEN pod args are enriched with service connection values", async () => {
+        await runEngineTestV5(
+          async ({ wsRoot }) => {
+            const cmd = new ExportPodV2CLICommand();
+            const podId = "dendron.gdoc";
+            const connectionId = "gdoc-main";
+            const configPath = getCustomConfigPath(wsRoot, podId);
+            const svcCongingPath = getSvcConfigPath(wsRoot, connectionId)
+            writeYAML(configPath, {
+              exportScope: "Workspace",
+              podType: PodV2Types.GoogleDocsExportV2,
+              connectionId
+            });
+            writeYAML(svcCongingPath, {
+              accessToken: "test",
+              refreshToken: "refresh",
+              expirationTime: Time.now(),
+              serviceType: ExternalService.GoogleDocs
+            });
+            const resp = await cmd.enrichArgs({
+              wsRoot,
+              podId,
+            });
+            expect(resp.data?.config.accessToken).toEqual("test");
+            expect(resp.data?.config.podType).toEqual(PodV2Types.GoogleDocsExportV2);
+            return;
+          },
+          { expect }
+        );
+      });
+    });
+});
```

Comment:
Great work with the tests, would it be useful to also add some tests for tests with invalid podIds or bad inline configs?